src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -242,133 +240,109 @@ export async function poll(
           console.info("Subscriber already notified, skipping: ", subscriberId);
           continue;
         }
-        const { recipientEmail, breachedEmail, signupLanguage } =
+        const { recipientEmail, breachedEmail } =
           getAddressesAndLanguageForEmail(recipient);
 
-        /* c8 ignore start */
-        const requestedLanguage = signupLanguage
-          ? acceptedLanguages(signupLanguage)
-          : [];
-        /* c8 ignore stop */
-
-        const availableLanguages = process.env.SUPPORTED_LOCALES!.split(",");
-        const supportedLocales = negotiateLanguages(
-          requestedLanguage,
-          availableLanguages,
-          { defaultLocale: "en" },
-        );
-
-        await localStorage.run(new Map(), async () => {
-          localStorage.getStore().set("locale", supportedLocales);
```

Comment:
Note: all the "new code" is just the code inside this `async` function with two fewer levels of indentation, but otherwise unchanged. So the only real change in this PR is the removal of lines 248-262.